modules/installation-user-infra-generate-k8s-manifest-ignition.adoc

```diff
@@ -61,6 +66,53 @@ Because you create and manage the worker machines yourself, you do not need
 to initialize these machines.
 endif::[]
 
+ifdef::gcp[]
+. Optional: If you do not want the cluster to provision compute machines, remove
+the Kubernetes manifest files that define the worker machines:
++
+----
+$ rm -f openshift/99_openshift-cluster-api_worker-machineset-*.yaml
+----
++
+Because you will create and manage the worker machines yourself, you do not need
+to initialize these machines.
+
+. Optional: Currently, emptying the compute pools makes control plane machines
```

Comment:
Is this really optional?
We'll need to untangle the conflict with https://github.com/openshift/openshift-docs/pull/17016/files